hpssa/hpssa.py

```diff
@@ -70,7 +70,12 @@ def parse_adapter_details(raw_data):
 
         if l[:3] != detail_indent:  # ascii space
             LOG.debug('Parsing array line: %s' % l)
-            name, slot = l.split('in Slot')
+            try:
+                name, slot = l.split('in Slot')
```

Comment:
From the reference show config data, this would still succeed, no?

Reply:
oh yes.. until it gets to the next line.. then it will fail.. Can we test for  CACHE STATUS PROBLEM DETECTED and ensure that gets recorded in the data structure? I'd prefer to capture that.. and set a new global error state, that would basically disable all other hpssa functions, but still record it